start.go

```diff
@@ -105,6 +105,14 @@ func SetGaugeWithLabels(key []string, val float32, labels []Label) {
 	globalMetrics.Load().(*Metrics).SetGaugeWithLabels(key, val, labels)
 }
 
+func SetPrecisionGauge(key []string, val float64) {
+	globalMetrics.Load().(*Metrics).SetPrecisionGauge(key, val)
+}
+
+func SetPrecisionGaugeWithLabels(key []string, val float64, labels []Label) {
+	globalMetrics.Load().(*Metrics).SetPrecisionGaugeWithLabels(key, val, labels)
+}
+
```

Comment:
This file in general is not well commented but I would lean towards adding doc comments for these functions. Partly because they form part of the libraries core API but specifically because they might panic if used with an incompatible sink and I think callers should be very clear about that before using them!